helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -11,6 +11,11 @@ extensions:
     directory: {{ .Values.logsCollection.checkpointPath }}
   {{- end }}
 
+  {{- if (eq (include "splunk-otel-collector.persistentQueueEnabled" .) "true") }}
```

Comment:
so we define both logs and metrics extensions when persistent queue is enabled, regardless if only logs or metrics are enabled. Not sure why?